.git-config/hooks/pre-commit

```diff
@@ -2,6 +2,7 @@
 LOCAL_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
 
 RENOVATE_PREFIX="^renovate/"
+GITHUB_REVERT_PREFIX="^revert-[0-9]{1,5}.*"
```

Comment:
Do we explicitly name any branch with revert prefix for any reasons apart from git created branches?